qa-tests-backend/src/test/groovy/ExternalNetworkSourcesTest.groovy

```diff
@@ -44,7 +42,19 @@ class ExternalNetworkSourcesTest extends BaseSpecification {
         return deployment
     }
 
+    @Shared
+    private int mask = 0
+
+    private String getBigCIDR() {
+        return "$CF_IP_ADDRESS/${20+mask}"
+    }
+
+    private String getSmallCIDR() {
+        return "$CF_IP_ADDRESS/${30-mask}"
+    }
+
     def setup() {
+        mask++
```

Comment:
Curious: would skipping this line cause those tests to fail again?

Reply:
Yes, I added this to have a different CIDR for every test case. There is a chance that test will not fail but it could be flaky